### ISSUE TITLE
Feature request: support sub-folders

### ISSUE THREAD

**Issue by perlun** [felipecaputo/git-project-manager#34]
Hi,

Thanks for a great VSCode extension. I have an issue with it, and the issue is that my git folders are extremely numerous (between 150 and 200), so I have split them per customer.

However, I cannot seem to be able to specify the "parent folder" which contains all these git repos. If I do, it will not find any repos. Any way to work around this?

**Comment by felipecaputo** [comment]
Hi @perlun, I'm glad you enjoy the extension

It will help if I create another command that allows you to choose which of the configured folders in `gitProjectManager.baseProjectsFolders` it will search, e.g. GPM - Show git projects from folder

This command will first list the configured folders in the configurarion, and after that list only projects that are into that folder.

What do you think?

**Comment by felipecaputo** [comment]
@perlun Sorry for taking too long. The feature is already implemented and I'm finalizing some tests to put it live today.

I hope it will help you!

**Comment by perlun** [comment]
I went with this approach for now:

```json
    "gitProjectManager.baseProjectsFolders": [
        "~/git/ecraft",
        "~/git/customer1",
        "~/git/customer2"
    ],
```

**Comment by felipecaputo** [comment]
I think I've had a bad understanding of your issue, maybe you can only set the `maxDepthRecursion` configuration to be higher, it means how much directories deep it can crawl.

The default value is 4, but you can increase it, the trade of is that the search may be slower

The new functionality I've build allows you to choose one of the base projects folder to list the projects

**Comment by felipecaputo** [comment]
@perlun New version released 1.2.0, hope it helps you!

**Comment by perlun** [comment]
Thanks @felipecaputo, appreciated!

My use case is: my `~/git` folder has very many folders in it. So its grouped in subfolders. If I set `gitProjectManager.baseProjectsFolders` to the _parent_ folder (which doesn't have any git folders in it), it didn't work, I had to list the specific folders under there. This was a bit inconvenient but it worked.
